### PR TITLE
sys-fs/xfsprogs: Fix build on musl 1.2.4

### PR DESCRIPTION
**Pull request by vimproved**
Closes: https://bugs.gentoo.org/907039

### CODE REVIEW

File: sys-fs/xfsprogs/files/xfsprogs-6.4.0-lfs64.patch [hunk 1]
```diff
@@ -0,0 +1,645 @@
+From a283105bee3de7941244a75d35379cb22a3efbbb Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline sys-fs/xfsprogs/files/xfsprogs-6.4.0-lfs64.patch:1]: Looks good, but upstream link please? (always!)

Comment (orbea) [reply sys-fs/xfsprogs/files/xfsprogs-6.4.0-lfs64.patch:1]: Its from Alpine.

https://git.alpinelinux.org/aports/tree/main/xfsprogs/lfs64.patch?id=0c57de04936543b82b55b320951b3ed84f3d45fb

I shared it on #xfs @ OFTC some while ago where it was being considered if it was the correct fix or not, but I never heard what happened after that.

Comment (thesamesam) [reply sys-fs/xfsprogs/files/xfsprogs-6.4.0-lfs64.patch:1]: This patch has her name on and includes justification for why glibc is fine with this change too. It's also the obviously-correct set of changes for the problem. That doesn't mean it's from Alpine.

Comment (vimproved) [reply sys-fs/xfsprogs/files/xfsprogs-6.4.0-lfs64.patch:1]: > Looks good, but upstream link please? (always!)

Where is the correct place to submit a patch for xfsprogs? I was looking at the XFS mailing list but I can't tell if that's the correct place for xfsprogs patches.

Comment (thesamesam) [reply sys-fs/xfsprogs/files/xfsprogs-6.4.0-lfs64.patch:1]: yeah, it really is linux-xfs, I sent a patch for xfsprogs last month: https://lore.kernel.org/linux-xfs/20230626095048.1290476-1-sam@gentoo.org/T/#u

Comment (akhuettel) [reply sys-fs/xfsprogs/files/xfsprogs-6.4.0-lfs64.patch:1]: Ping. Any success with upstreaming?

**Comment by orbea** [comment]
@vimproved Looks like this is now applied in Gentoo?

https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=d64e0a8bf2551bcd233b11faca7cc49d01b10bc2

**Pull request closed without merging**